source/reference/minio-mc/mc-support-inspect.rst

```diff
@@ -0,0 +1,139 @@
+======================
+``mc support inspect``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support inspect
+
+Description
+-----------
+
+The :mc-cmd:`mc support inspect` command collects the data and metadata associated to objects at the specified path.
+MinIO assembles this data from each backend drive storing an :ref:`erasure shard <minio-erasure-coding>` for each specified object.
+
+The command produces a zip file that includes all matching files with their respective *host+drive+path*. 
+
+You can export the contents to a JSON output for further analysis.
+
+The resulting report is intended for use by MinIO Engineering via |SUBNET| and may contain internal or private data points.
+MinIO anonymizes hostnames and IP addresses before sending or storing the report.
+Exercise caution before sending a report to a third party or posting the report in a public forum.
+
+.. note::
+   
+   :mc-cmd:`mc support inspect` requires a MinIO deployment server from October 2021 or later. 
+
+Wildcards
+---------
+
+The command allows for the wildcard character, ``*``. 
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support inspect ALIAS/bucket/path/**/xl.meta
+   
+This command collects all ``xl.meta`` recursively.
+
+
+Examples
+--------
+
+Download Metadata for an Object
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can download the metadata for an object.
+Metadata stores in an ``xl.meta`` binary file.
+
+- MinIO alias: ``minio1``
+- Bucket: ``mybucket``
+- Object: ``myobject``
```

Comment:
Style nit: We can paragraphize this as:

"The following command downloads the xl.meta from ``mybucket/myobject`` on the ``minio1`` MinIO deployment

Or if we want list-style ,convert the example to use placeholders and do the list after:

```
.. code-block:: shell

   mc support inspect ALIAS/BUCKET/OBJECT/xl.meta

- Replace ``ALIAS`` with the :ref:`alias <alias>` for the MinIO deployment
- Replace ``BUCKET`` with the bucket...
- Replace ``OBJECT`` with the full path to the object...

Reply:
👍